app/views/ctc/questions/advance_ctc_received/edit.html.erb

```diff
@@ -10,7 +10,7 @@
       <h2><%= t("views.ctc.questions.advance_ctc_received.already_received") %></h2>
       <%= link_to t("general.edit").downcase, questions_advance_ctc_path, class: "review-box__edit review-box__edit-button" %>
     </div>
-    <p class="spacing-below-15"><%= t("views.ctc.questions.advance_ctc_received.total_adv_ctc", amount: current_intake.advance_ctc_amount_received) %></p>
+    <p class="spacing-below-15"><%= t("views.ctc.questions.advance_ctc_received.total_adv_ctc", amount: calculated_or_provided_dollar_amount(current_intake.advance_ctc_amount_received)) %></p>
```

Comment:
Unrelated to this pull request: Can we remove the `fallback_value` from calculated_or_provided_dollar_amount?